R/semantic_search.R

```diff
@@ -0,0 +1,105 @@
+#' define_selection_type
+#'
+#' @param name - string with object name
+#' @param multiple - logical
+define_selection_type <- function(name, multiple) {
+  if (multiple == T) {
+    paste("ui fluid multiple search selection dropdown", name)
+  } else {
+    paste("ui fluid search selection dropdown", name)
+  }
+}
+
+#' semantic_search_api
+#'
+#' function defining the (multiple) search selection dropdown input using REST API
+#'
+#' @param name string with input name
```

Comment:
this is difficult for me since i don;t fully understand this that happen here :) so maybe you could help me with that and explain :)?